.github/workflows/build.yml

```diff
@@ -0,0 +1,27 @@
+name: Build cwms-radar-client
+on:
+  push:
+    branches:
+      - main
```

Comment:
thanks for catching that typo, it should have been "feature" instead of "release". I definitely don't want to leave this on main, otherwise all of the feature branches won't get fast compile/test feedback. Additionally, the DevOps pieces so far have need a build triggered in order to test out functionality before merging into main.